kisanola/tech-community-frontend
# Complete the display of the profile

#### What does this PR do?
Adds the complete display of the profile page and improve the layout of the 404 page 

#### Description of Task to be completed?
* Add the `links` part and the `Following` part on the profile page
* Design the 404 page 

#### How should this be manually tested?
After being logged into the platform, head over to the profile page and notice the difference. Add some whatever string to the url and notice the new 404 page
